src/settings.py

```diff
@@ -67,4 +67,11 @@ class TokenSettings:
     "STRK-USDC",
     "STRK-USDT",
     "STRK-DAI",
-]
\ No newline at end of file
+]
+
+COLLATERAL_TO_DEBT_TOKENS = {}
```

Comment:
Pls specify `COLLATERAL_TOKENS` and `DEBT_TOKENS` separately (just 3 values for each now). I don't expect the debt tokens to change based on which collateral token was selected.

Reply:
If we don't change the debt token based on the collateral token, the application will have an error while compiling